examples/nidigital/configure-voltage-levels-and-termination-voltage.py

```diff
@@ -0,0 +1,200 @@
+# This example demonstrates how to use the NI-Digital Pattern Driver API to create an
+# instrument session, configure various voltage levels, and then burst a
+# pattern. The vol, voh, vih, vil, iol, ioh and vcom values are configurable parameters and the
+# user can select from High-Z (h), Active Load (a) and Three-Level Drive (t)
+# termination options in termSelect.
+#
+# The gRPC API is built from the C API.  NI-Digital Pattern Driver Pattern documentation is installed with
+# the driver at:
+# C:\Users\Public\Documents\National Instruments\NI-Digital-Pattern-Driver\Documentation\Digital Pattern Help.chm
+#
```

Comment:
Add a line "Copy the  .pinmap, .specs, .digitiming & .digipat files that come with this example to a folder on the server machine & note down the path of the folder".